samples/bot-teams-authentication/nodejs/config.js

```diff
@@ -0,0 +1,8 @@
+const config = {
+  MicrosoftAppId: process.env.CLIENT_ID,
```

Comment:
these are the config values from BotFramework, instead we should use our newer configuration syntax